### DISCUSSION TITLE
Discussions moving to Discord

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#762] [Announcements]
Hi Community! 

We created a discord server for all things Leantime and Startups:

The invite link is here: [Leantime Discord](https://discord.gg/4zMzJtAq9z)

Looking forward to chatting!

**Comment by marcelfolaron** [comment]
We have decided to move the discussion boards to Discord alltogether. There are more community members and you will get an answer quicker. Please consider the discussion board here archived

**Comment by fpvuqhkfua** [comment]
Hi Friends,

We have a local deployment of the leantime application, suddenly the application webpage started returning   "No database connection, check your database credentials in your configuration file." I checked the .env file in .dev folder at the base machine, can someone guide me what could have gone wrong, the main issue is the architect who deployed the application is not available, hence I am doing hit ant trial. there an env file in the root folder project and a .env file inside the leantime-dev container.
